### DISCUSSION TITLE
Expiring Sessions.

### DISCUSSION THREAD

**Discussion by aarondyejr** [maxcountryman/axum-login#236] [Q&A] [answered]
Hello all! I am inquiring on how to make it so sessions don't expire, but rather only when they logout are they no longer logged in. My project requires that sessions remain active until they logout themselves.

**Comment by maxcountryman** [comment, marked as answer]
You can [set an expiry](https://docs.rs/tower-sessions/latest/tower_sessions/enum.Expiry.html) at a date far in the future, such that the only practical way a user would be logged out is if they log themselves out.

**Reply by aarondyejr** [reply]
Thank you so much!

**Reply by aarondyejr** [reply]
@maxcountryman is there a dedicated community for this project, or is it all done through this repo?

**Reply by maxcountryman** [reply]
For now everything is on GitHub Discussions.

**Reply by frederikhors** [reply]
And is very good! ♥
